packages/front/svelte/views/auth/login/Login.svelte

```diff
@@ -10,33 +12,94 @@
     $: controller.formElt = form;
 </script>
 
-<div class="fr-container fr-mb-8w">
-    <div class="fr-grid-row fr-grid-row--center fr-grid-row--gutters">
-        <div class="fr-col fr-col-lg-8">
-            <h1>Connexion</h1>
-            {#if $error != null}
-                <Alert title="Attention">
-                    {$error}
-                </Alert>
-            {/if}
-            {#if showSuccessMessage}
-                <Alert type="success">
-                    {successMessage}
-                </Alert>
-            {/if}
-            <form bind:this={form} on:submit={e => controller.submit(e)}>
-                <fieldset class="fr-fieldset fr-my-4w">
-                    <legend class="fr-fieldset__legend" id="text-legend">Vos identifiants</legend>
-                    <label class="fr-label fr-pl-1w" for="text-input-text">Email :</label>
-                    <input class="fr-input fr-ml-1w" type="email" name="email" required="true" />
-                    <label class="fr-label fr-mt-2w fr-ml-1w" for="text-input-text">Mot de passe :</label>
-                    <input class="fr-input fr-ml-1w" type="password" name="password" required="true" />
-                </fieldset>
-                <div class="fr-input-group fr-my-4w">
-                    <button type="submit" class="fr-btn" title="Connection">Valider</button>
-                    <a class="fr-link" href="/auth/forget-password">Mot de passe oublié</a>
+<h1 class="fr-mb-6w fr-h2">
+    {controller.pageTitle}
+</h1>
+{#if $error != null}
+    <Alert title="Attention">
+        {$error}
+    </Alert>
+{/if}
+{#if showSuccessMessage}
+    <Alert type="success">
+        {successMessage}
+    </Alert>
+{/if}
+<div class="login fr-col-6 fr-col-offset-3 fr-p-8v fr-mt-7v">
+    <form on:submit|preventDefault={() => controller.submit()}>
+        <fieldset class="fr-fieldset fr-mb-5w">
+            <legend class="fr-fieldset__legend" id="text-legend">Se connecter à son compte</legend>
+            <div class="fr-fieldset__element fr-mt-4v">
+                <Input
+                    type="email"
+                    label="Email professionnel"
+                    id="email-input"
+                    bind:value={controller.email}
```

Comment:
```suggestion
                    id="email-input"
                    autocomplete="email"
                    bind:value={controller.email}
```
J'ai bien aimé l'autocomplete plus bas alors autant le mettre ici aussi